addons/mail/static/src/models/user_setting.js

```diff
@@ -260,31 +285,35 @@ registerModel({
             inverse: 'userSetting',
             isCausal: true,
         }),
+        globalSettingsTimeout: attr(),
         /**
-         * true if the user wants to use push to talk (over voice activation)
+         * If true, push-to-talk will be used over voice activation.
          */
-        use_push_to_talk: attr({
+        isPushToTalkUsed: attr({
+            compute: '_computeIsPushToTalkUsed',
```

Comment:
Diff is hard to read because the old field name has been moved to new model, and here it's a rename and becomes related.

It would be much easier to split in smaller commit with small and very clear concern. For example, one commit should introduce the model (just the model, nothing else), then another commit to move the fields, then another commit to introduce compute fields from `UserSetting` to `res.users.settings`, etc.

This is maybe more diff overall, but it's significantly much easier and safer to review than current diff in a single commit.